apps/src/templates/manageStudents/ManageStudentsGenderCell.jsx

```diff
@@ -0,0 +1,50 @@
+import React, {Component, PropTypes} from 'react';
+import i18n from "@cdo/locale";
+
+const GENDERS = {
+  m: i18n.genderMale(),
+  f: i18n.genderFemale()
+};
+
+class ManageStudentGenderCell extends Component {
+  static propTypes = {
+    id: PropTypes.number.isRequired,
+    gender: PropTypes.string,
+    isEditing: PropTypes.bool,
+  };
+
+  state = {
+    genderValue: this.props.gender
+  };
+
+  onChangeGender = (e) => {
+    this.setState({genderValue: e.target.value});
+  }
+
+  render() {
+    const genders = ['m', 'f'];
```

Comment:
Could be `const genders = Object.keys(GENDERS);`, which I think makes sense.

It looks like you only use it in the one place too, so I might just on line  42 do
```jsx
Object.keys(GENDERS).map(gender => 
  <option key={gender} value={gender}>
    {GENDERS[gender]}
  </option>
)
```

Reply:
Do we not support stating "Other" for gender?